cmd/rekor_monitor/main.go

```diff
@@ -76,12 +69,16 @@ func main() {
 		if err := yaml.Unmarshal([]byte(configString), &config); err != nil {
 			log.Fatalf("error parsing identities: %v", err)
 		}
+
+		configProvided = true
 	}
 
 	if *configYamlInput != "" {
 		if err := yaml.Unmarshal([]byte(*configYamlInput), &config); err != nil {
 			log.Fatalf("error parsing identities: %v", err)
 		}
+
+		configProvided = true
 	}
 
 	if config.OutputIdentitiesFile == "" {
```

Comment:
Below for `VerifyConsistencyCheckInputs`, given we provide defaults, do we need this check anymore? Just need to update `config.LogINfoFile` based on the default if it's not provided.

Reply:
updating logInfoFile and removing this part of the check